daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/Transaction.scala

```diff
@@ -813,6 +814,7 @@ object Transaction {
       usedPackages: Set[PackageId],
       dependsOnTime: Boolean,
       nodeSeeds: ImmArray[(NodeId, crypto.Hash)],
+      disclosures: ImmArray[DisclosedContract],
```

Comment:
We need `Versioned[DisclosedContract]` here.

Reply:
just changed it.